src/plugins/ssu_catch.rs

```diff
@@ -325,6 +325,7 @@ mod tests {
         let first_post_metadata = &posts_metadata[0];
 
         // 실제 게시물 가져오기
+        #[allow(clippy::needless_borrow)]
         let post = ssu_catch_plugin
             .fetch_post(&first_post_metadata)
```

Comment:
왜 이걸 못봤지.. 아래처럼 수정해 주실 수 있을까요
```suggestion
let post = ssu_catch_plugin
            .fetch_post(first_post_metadata)
```

Reply:
반영했습니다! https://github.com/yourssu/ssufid/pull/79/commits/e649edd429b247aadf81a2665a79b280e4928ac4